draft-ietf-quic-reliable-stream-reset.md

```diff
@@ -171,6 +171,11 @@ Reliable Size, the receiver only needs to deliver data up the lower Reliable
 Size to the application. It MUST NOT expect the delivery of any data beyond that
 byte offset.
 
+After sending multiple frames for the same stream, the state transition to the
+"Reset Recvd" state only occurs when receiving the acknowledgement for the frame
+with the frame carrying the lowest Reliable Size. Acknowledgements for
+RESET_STREAM_AT frames with higher offsets don't cause a state transition.
```

Comment:
I'd be wary of calling out the terminal state here. The problem is that we will be waiting for all the acks for STREAM frames up to Reliable Size, that leads us to "Data Recvd" not "Reset Recvd."

But if we want to talk about the terminal state using names, I think we'd need to describe the entire state transition, because I can see people getting confused with a RESET_STREAM_AT frame leading to "Data Recvd" on the sender side, "Data Read" on the client side.

For the purpose of this pull request, I think it might be a good idea to lightly state that the sender MUST ensure delivery of an RESET_STREAM_AT frame carrying the smallest Reliable Size, possibly in the first paragraph of this subsection. That is where we talk about the sender reducing Reliable Size of the RESET_STREAM_AT frame.